matchers/matchers.go

```diff
@@ -118,11 +120,23 @@ var supportedMatchers = map[string]*matcherFactory{
 	".py": pythonMatcherFactory,
 }
 
+// Matchers for comments and todo strings
+type Matchers struct {
+	todos []string
+}
+
+// NewMatchers returns Matcher instance
+func NewMatchers(customTodos []string) *Matchers {
+	return &Matchers{
+		todos: customTodos,
+	}
+}
```

Comment:
Instead of creating a `Matchers` struct here just to hold the custom todos, you can add an additional field in the `checker` and pass along the custom todos on each call to `TodoMatcherForFile `

Reply:
Fixed